### PR TITLE
feat(admin): configurable tabs via admin config array

### LINKED ISSUE #3846: feat(admin): configurable tabs via admin config array
## Problem

The admin page (`/admin`) has hardcoded tabs (Users, Organizations). Downstream projects need to add their own admin tabs (e.g. Costs, Logs, Billing) but currently have to either:
- Fork `admin.view.vue` and hardcode new tabs
- Create separate sidenav items (bad UX — splits admin into multiple nav entries)

## Proposed solution

Add an `adminTabs` array to the admin module config (or the app-level config) that downstream projects can extend:

\`\`\`js
// config/index.js (downstream project)
admin: {
  tabs: [
    { value: 'costs', label: 'Costs', icon: 'fa-solid fa-coins', route: '/admin/costs' },
  ],
}
\`\`\`

`admin.view.vue` reads this config and renders extra tabs alongside the default Users/Organizations tabs.

## Expected behavior

- Default tabs (Users, Organizations) always rendered
- Extra tabs from config rendered after defaults
- Each extra tab is a `<router-link>` / `<v-tab :to="tab.route">` pointing to the downstream route
- Tab hidden if user doesn't have `manage UserAdmin` ability (same guard as today)
- No change to existing downstream projects that don't use this config

## Acceptance criteria

- [ ] `config.admin.tabs` array read in `admin.view.vue`
- [ ] Each entry: `{ value, label, icon, route }` — `icon` optional
- [ ] Tabs rendered as `v-tab` with `<router-link>` (or `to` prop)
- [ ] Falls back gracefully to current behavior when array is empty/undefined
- [ ] Unit test: renders extra tab when config provides one
- [ ] Downstream usage example in devkit docs or comment

## Context

First downstream use case: `trawl_vue` needs a "Costs" tab → `/admin/costs` (cost tracking per history — D5.5).

### PR DESCRIPTION
**Pull request by PierreBrisorgueil**
## Summary
- Add `config.admin.tabs` array support in `admin.view.vue` to render extra `v-tab` entries after the default Users and Organizations tabs
- Each entry: `{ value, label, icon?, route }` — icon is optional, route sets the `to` prop
- Graceful fallback when array is empty or undefined (current behavior preserved)
- Add 5 unit tests covering: default tabs, extra tabs with icon, extra tabs without icon, empty array, missing config
- Add `admin.tabs` to development config defaults with documented example

## Test plan
- [x] Unit tests: 5 new tests in `admin.view.unit.tests.js` (all pass)
- [x] Full test suite: 749 tests pass (57 files)
- [x] Lint: zero errors
- [ ] CI green

Closes #3846

### CODE REVIEW

File: src/modules/admin/views/admin.view.vue [hunk 1]
```diff
@@ -165,6 +175,14 @@ export default {
     };
   },
   computed: {
+    /**
+     * @desc Extra admin tabs from config.admin.tabs array.
+     * Each entry: { value, label, icon?, route }
+     * @returns {Array}
+     */
+    extraTabs() {
+      return this.config?.admin?.tabs || [];
```
[the remark this answers is not in the document]
Comment (PierreBrisorgueil) [reply src/modules/admin/views/admin.view.vue]: Fixed in 96f2561e: added `Array.isArray` guard and `.filter()` to validate each entry has required fields (`value`, `label`, `route`). Invalid entries (null, non-objects, missing fields) are now silently filtered out.

File: src/modules/admin/tests/admin.view.unit.tests.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      admin: {
+        tabs: [
+          { value: 'logs', label: 'Logs', route: '/admin/logs' },
+        ],
+      },
+    });
+    const tabs = wrapper.findAllComponents({ name: 'VTab' });
+    expect(tabs.length).toBe(3);
+    expect(tabs[2].text()).toContain('Logs');
+  });
```
[the remark this answers is not in the document]
Comment (PierreBrisorgueil) [reply src/modules/admin/tests/admin.view.unit.tests.js]: Fixed in 96f2561e: added VIcon presence/absence assertions — test with icon verifies icon component exists with correct prop, test without icon asserts zero VIcon children. Also added tests for non-array config and missing required fields.
